### PR TITLE
Fix typos in the projection code list

### LINKED ISSUE #583: 投影代码列表
## 问题描述

投影代码列表汇总中有些投影方式的 *width* 参数前缺少 `/`. 见 https://docs.gmt-china.org/latest/proj/.

需要修改 https://github.com/gmt-china/GMT_docs/blob/master/source/option/projection-codes.rst_.

### PR DESCRIPTION
**Pull request by ZMAlt**
Fixes #583

### CODE REVIEW

File: source/option/projection-codes.rst_ [hunk 1]
```diff
@@ -15,21 +15,21 @@
...
    | **-JF**\ |lon0|/|lat0|\ [/\ *horizon*]/\ *width*         | Azimuthal gnomonic                                          |
    +----------------------------------------------------------+-------------------------------------------------------------+
    | **-JG**\ |lon0|/|lat0|\ [/\ *horizon*]/\ *width*         | Azimuthal orthographic                                      |
    +----------------------------------------------------------+-------------------------------------------------------------+
-   | **-JG**\ |lon0|/|lat0|\                                  |                                                             |
+   | **-JG**\ |lon0|/|lat0|/\                                 |                                                             |
    | *alt*/*azim*/*tilt*/*twist*/*W*/*H/width*                | General perspective                                         |
    +----------------------------------------------------------+-------------------------------------------------------------+
    | **-JH**\ [|lon0|/]\ *width*                              | Hammer equal area                                           |
```
Comment (core-man) [inline source/option/projection-codes.rst_:35]: GMT 官方手册 (https://docs.generic-mapping-tools.org/latest/cookbook/map-projections.html):
- Hammer equal area
- Sinusoidal equal area 
- Miller cylindrical 
- Eckert IV equal area 
- Eckert VI equal area

> lon0/scale|width

其实应该是 [lon0/]scale|width

File: source/option/projection-codes.rst_ [hunk 2]
```diff
@@ -42,29 +42,31 @@
...
    +----------------------------------------------------------+-------------------------------------------------------------+
-   | **-JOa**\ |lon0|/|lat0|\ *azim*/*width*                  | Oblique Mercator, 1: origin and azim                        |
+   | **-JOa**\ |lon0|/|lat0|/\ *azim*/*width*                 | Oblique Mercator, 1: origin and azim                        |
    +----------------------------------------------------------+-------------------------------------------------------------+
-   | **-JOb**\ |lon0|/|lat0|/|lon1|/|lat1|\ *width*           | Oblique Mercator, 2: two points                             |
+   | **-JOb**\ |lon0|/|lat0|/|lon1|/|lat1|/\ *width*          | Oblique Mercator, 2: two points                             |
    +----------------------------------------------------------+-------------------------------------------------------------+
-   | **-JOc**\ |lon0|/|lat0|/|lonp|/|latp|\ *width*           | Oblique Mercator, 3: origin and pole                        |
+   | **-JOc**\ |lon0|/|lat0|/|lonp|/|latp|/\ *width*          | Oblique Mercator, 3: origin and pole                        |
```
Comment (core-man) [inline source/option/projection-codes.rst_:55]: 加上 **+v** 选项？

中文手册和官方手册都有提到 **+v** 选项：
- https://docs.gmt-china.org/latest/proj/Jo/
- https://docs.generic-mapping-tools.org/latest/cookbook/map-projections.html#oblique-mercator-projection-jo-jo

---

另外 GMT 官方手册里 **width** 后多了一个空格：

`Oblique Mercator, 1: origin and azim | -Jo\|O[a\|A] | -Jomerc/ | lon0/lat0/azim/scale\|width [+v]`

File: source/option/projection-codes.rst_ [hunk 3]
```diff
@@ -15,21 +15,21 @@
...
    | **-JE**\ |lon0|/|lat0|\ [/\ *horizon*]/\ *width*         | Azimuthal equidistant                                       |
    +----------------------------------------------------------+-------------------------------------------------------------+
    | **-JF**\ |lon0|/|lat0|\ [/\ *horizon*]/\ *width*         | Azimuthal gnomonic                                          |
    +----------------------------------------------------------+-------------------------------------------------------------+
    | **-JG**\ |lon0|/|lat0|\ [/\ *horizon*]/\ *width*         | Azimuthal orthographic                                      |
    +----------------------------------------------------------+-------------------------------------------------------------+
-   | **-JG**\ |lon0|/|lat0|\                                  |                                                             |
+   | **-JG**\ |lon0|/|lat0|/\                                 |                                                             |
    | *alt*/*azim*/*tilt*/*twist*/*W*/*H/width*                | General perspective                                         |
```
Comment (core-man) [inline source/option/projection-codes.rst_:33]:
```suggestion
   | *alt*/*azim*/*tilt*/*twist*/*W*/*H*/*width*              | General perspective                                         |
```

File: source/option/projection-codes.rst_ (hunk 1 above)
Comment (ZMAlt) [reply source/option/projection-codes.rst_:35]: 先记下，有空去 commit

File: source/option/projection-codes.rst_ (hunk 2 above)
Comment (ZMAlt) [reply source/option/projection-codes.rst_]: 加 **+v** 了

**Review by core-man — APPROVED** [review]
Great work~

**Pull request merged**